src/main/java/com/team701/buddymatcher/controllers/communication/Communication.json

```diff
@@ -0,0 +1,48 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "backend API",
+    "description": "backend API",
+    "version": "1.0.0"
+  },
+  "servers": [
+    {
+      "url": "https://backend"
+    }
+  ],
+  "paths": {
+    "/api/communication/messages": {
+      "post": {
+        "summary": "POST api/communication/messages",
+        "operationId": "sendMessage",
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        }
+      }
+    },
```

Comment:
This endpoint is being removed in #72

Reply:
Ok I have removed it let me know if there is anything you think I should add or change